### PR TITLE
Fix #1059 Move code from devoxx be 19 project to core

### LINKED ISSUE #1059: Move Devoxx 2019 BE code to core
**Is your feature request related to a problem? Please describe.**
Devoxx 2019 BE code is copied over and over again. 

**Describe the solution you'd like**
To inhibit this and prepare the codebase for core 2.0 refactoring the code should be made more generic and moved to main repository

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by svenreimers**

### CODE REVIEW

File: devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/steps/ShowSchedule.java [hunk 1]
```diff
@@ -0,0 +1,276 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 TweetWallFX
```
Comment (mklaehn) [inline devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/steps/ShowSchedule.java:4]:
```suggestion
 * Copyright (c) 2022 TweetWallFX
```
or
```suggestion
 * Copyright (c) 2019-2022 TweetWallFX
```

File: generic2d/build.gradle [hunk 2]
```diff
@@ -27,4 +27,6 @@ dependencies {
     implementation project(':tweetwallfx-2d')
     implementation project(':tweetwallfx-core')
     implementation project(':tweetwallfx-tweet-api')
+    implementation project(':tweetwallfx-stepengine-steps')
+    implementation project(':tweetwallfx-tweet-impl-twitter4j')
```
Comment (mklaehn) [inline generic2d/build.gradle:31]: An API implementation as implementation dependency?

```suggestion
    runtimeOnly project(':tweetwallfx-tweet-impl-twitter4j')
```

File: devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/dataprovider/ScheduleDataProvider.java [hunk 3]
```diff
@@ -94,7 +94,7 @@ public DataProvider create(final StepEngineSettings.DataProviderSetting dataProv
     }
 
     /**
-     * POJO used to configure {@link TopTalksWeekDataProvider}.
+     * POJO used to configure {@link ScheduleDataProvider}.
```
Comment (mklaehn) [inline devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/dataprovider/ScheduleDataProvider.java:97]: Shouldn't the change require an update of the years in the header? Can be done when releasing though

**Review by mklaehn — COMMENTED** [review]
Overall positive. Some small changes to dependency handling though

**Review by mklaehn — DISMISSED** [review]

File: devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/dataprovider/ScheduleDataProvider.java (hunk 3 above)
Comment (svenreimers) [reply devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/dataprovider/ScheduleDataProvider.java:97]: Done

**Review by mklaehn — APPROVED** [review]

**Pull request merged**